README.md

```diff
@@ -547,46 +589,42 @@ creation of the underlying Cyphers.
 To use it in an existing SBT project, add the following dependency to your `build.sbt`:
 
 ```scala
-libraryDependencies += "com.velocidi" %% "apso-encryption" % "0.16.0"
+libraryDependencies += "com.velocidi" %% "apso-encryption" % "0.16.1"
 ```
 
 The following shows the creation of `Encryptor` and `Decryptor` objects,
 by loading a `KeyStore` file holding a symmetric key, and its use to encrypt and
 decrypt data:
 
 ```scala
-scala> val encryptor = Encryptor("AES", getClass.getResourceAsStream("/keystoreFile.jceks"), "keystorePass", "keyAlias", "keyPass")
-encryptor: Option[com.velocidi.apso.encryption.Encryptor] = Some(com.velocidi.apso.encryption.Encryptor@353912)
+import com.velocidi.apso.encryption._
 
-scala> val decryptor = Decryptor("AES", getClass.getResourceAsStream("/keystoreFile.jceks"), "keystorePass", "keyAlias", "keyPass")
-decryptor: Option[com.velocidi.apso.encryption.Decryptor] = Some(com.velocidi.apso.encryption.Decryptor@68ccfc03)
+val encryptor = Encryptor("AES", getClass.getResourceAsStream("/keystoreFile.jceks"), "keystorePass", "keyAlias", "keyPass")
 
-scala> val secretData = "secret_info"
-secretData: String = secret_info
+val decryptor = Decryptor("AES", getClass.getResourceAsStream("/keystoreFile.jceks"), "keystorePass", "keyAlias", "keyPass")
 
-// encrypt data and encode it in base64; then decrypt it to string
-scala> decryptor.get.decryptToString(encryptor.get.encryptToSafeString(secretData).get)
-res6: Option[String] = Some(secret_info)
+val secretData = "secret_info"
 
+// encrypt data and encode it in base64; then decrypt it to string
+decryptor.get.decryptToString(encryptor.get.encryptToSafeString(secretData).get)
 ```
 
 ## Hashing
 
 Apso provides utilities for various hashing functions. To use it in an existing SBT project, add the following dependency to your `build.sbt`:
 
 ```scala
-libraryDependencies += "com.velocidi" %% "apso-hashing" % "0.16.0"
+libraryDependencies += "com.velocidi" %% "apso-hashing" % "@VERSION"
```

Comment:
Can you run `docs/mdoc` so that this gets updated?

Reply:
Fixed on 4bf9493bac5ae546c0a4c79f0ee22bc9bde0a885.